### PR TITLE
Model updates

### PR DESCRIPTION
**Pull request by themightychris**

### CODE REVIEW

File: sencha-workspace/SlateAdmin/app/model/person/Relationship.js [hunk 1]
```diff
@@ -3,11 +3,14 @@ Ext.define('SlateAdmin.model.person.Relationship', {
     extend: 'Ext.data.Model',
     requires: [
 //        'Jarvus.ext.override.data.CallbackValidation',
+        'SlateAdmin.model.validator.CallbackValidator',
         'SlateAdmin.proxy.Records'
     ],
 
     idProperty: 'ID',
 
+    validationSeparator: 'jjjjjjjjjjjjj',
```
Comment (themightychris) [inline sencha-workspace/SlateAdmin/app/model/person/Relationship.js]: yuck, do we need this? I don't think we need to store multiple validation errors per field, the default behavior of only storing the first seems like what we want and have relied on in the past

File: sencha-workspace/SlateAdmin/app/model/person/Relationship.js [hunk 2]
```diff
@@ -59,37 +62,26 @@ Ext.define('SlateAdmin.model.person.Relationship', {
...
-        {
-            type: 'presence',
-            field: 'Class'
-        },
-        {
-            type: 'presence',
-            field: 'PersonID'
-        },
-        {
+    validators: {
```
Comment (themightychris) [inline sencha-workspace/SlateAdmin/app/model/person/Relationship.js]: see #13

File: sencha-workspace/SlateAdmin/app/model/person/ContactPoint.js [hunk 3]
```diff
@@ -81,17 +72,17 @@ Ext.define('SlateAdmin.model.person.ContactPoint', {
         include: ['String', 'Primary'],
         relatedTable: ['Person']
     },
-    
+
     validate: function() {
```
Comment (themightychris) [inline sencha-workspace/SlateAdmin/app/model/person/ContactPoint.js]: see #13

File: sencha-workspace/SlateAdmin/app/model/person/Relationship.js (hunk 1 above)
Comment (BillClinton) [inline sencha-workspace/SlateAdmin/app/model/person/Relationship.js]: no.  I was testing and inadvertently left it in there.  I have removed it.

File: sencha-workspace/SlateAdmin/app/model/validator/CallbackValidator.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
+/*jslint browser: true, undef: true *//*global Ext*/
+/**
+ * Validates that the passed value is not `null` or `undefined` or `''`.
+ */
+Ext.define('SlateAdmin.model.validator.CallbackValidator', {
```
Comment (themightychris) [inline sencha-workspace/SlateAdmin/app/model/validator/CallbackValidator.js]: @BillClinton let's move this up to just `SlateAdmin.validator.CallbackValidator`

File: sencha-workspace/SlateAdmin/app/model/person/Relationship.js [hunk 5]
```diff
@@ -3,6 +3,7 @@ Ext.define('SlateAdmin.model.person.Relationship', {
     extend: 'Ext.data.Model',
     requires: [
 //        'Jarvus.ext.override.data.CallbackValidation',
```
Comment (themightychris) [inline sencha-workspace/SlateAdmin/app/model/person/Relationship.js]: @BillClinton delete commented dependency since it's been replaced

**Pull request merged**